src/components/Footer/Footer.test.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+import { render, screen } from "@testing-library/react"
+import Footer from "./Footer"
+
+describe("Footer", () => {
+  it("renders Footer component", () => {
```

Comment:
tests are important, and should cover somethings in general like special chars, any kind of possible errors. and better to implement separate test for each case, not good to test different logic in one test.

Reply:
footer tests rewritten